google/cloud/aiplatform/schema.py

```diff
@@ -24,6 +24,7 @@ class definition:
         automl_tabular = "gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_tabular_1.0.0.yaml"
         automl_forecasting = "gs://google-cloud-aiplatform/schema/trainingjob/definition/automl_time_series_forecasting_1.0.0.yaml"
         seq2seq_plus_forecasting = "gs://google-cloud-aiplatform/schema/trainingjob/definition/seq2seq_plus_time_series_forecasting_1.0.0.yaml"
+        tft_forecasting = "gs://google-cloud-aiplatform/schema/trainingjob/definition/temporal_fusion_transformer_time_series_forecasting_1.0.0.yaml"
```

Comment:
@mikelawrence-google can we confirm this schema is published and can be accessed?

Reply:
Yep, looks good from my side.

```
$ gsutil cat gs://google-cloud-aiplatform/schema/trainingjob/definition/temporal_fusion_transformer_time_series_forecasting_1.0.0.yaml
type: TemporalFusionTransformerTimeSeriesForecasting
version: 1.0.0
description: >
  A training task that trains a Temporal Fusion Transformer (TFT) Forecasting
  Model via the
...
...
```